datasets/omp/omp.py

```diff
@@ -0,0 +1,263 @@
+
+For each article, the data set contains the following data:
+
+* Article ID
+* Publishing date
+* Topic Path (e.g.: Newsroom / Sports / Motorsports / Formula 1)
+* Title
+* Body
+
+Detailed descriptions of the post selection and annotation procedures are given in the paper.
+
+## Annotated Categories
+
+Potentially undesirable content:
+
+* Sentiment (negative/neutral/positive)
+    An important goal is to detect changes in the prevalent sentiment in a discussion, e.g.,
+    the location within the fora and the point in time where a turn from positive/neutral
+    sentiment to negative sentiment takes place.
+* Off-Topic (yes/no)
+    Posts which digress too far from the topic of the corresponding article.
+* Inappropriate (yes/no)
+    Swearwords, suggestive and obscene language, insults, threats etc.
+* Discriminating (yes/no)
+    Racist, sexist, misogynistic, homophobic, antisemitic and other misanthropic content.
+
+Neutral content that requires a reaction:
+
+* Feedback (yes/no)
+    Sometimes users ask questions or give feedback to the author of the article or the
+    newspaper in general, which may require a reply/reaction.
+
+Potentially desirable content:
+
+* Personal Stories (yes/no)
+    In certain fora, users are encouraged to share their personal stories, experiences,
+    anecdotes etc. regarding the respective topic.
+* Arguments Used (yes/no)
+    It is desirable for users to back their statements with rational argumentation,
+    reasoning and sources.
+"""
+
+_HOMEPAGE = "https://ofai.github.io/million-post-corpus/"
+
+_LICENSE = "Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International License"
+
+_URLs = {
+    "posts_labeled": "https://github.com/aseifert/million-post-corpus/raw/master/data/posts_labeled.csv.xz",
+    "posts_unlabeled": "https://github.com/aseifert/million-post-corpus/raw/master/data/posts_unlabeled.csv.xz",
+    "articles": "https://github.com/aseifert/million-post-corpus/raw/master/data/articles.csv.xz",
+}
+
+
+class Omp(datasets.GeneratorBasedBuilder):
+    """The “One Million Posts” corpus is an annotated data set consisting of user comments
+    posted to an Austrian newspaper website (in German language). Annotated categories include:
+    sentiment (negative/neutral/positive), off-topic (yes/no), inappropriate (yes/no),
+    discriminating (yes/no), feedback (yes/no), personal story (yes/no), arguments used (yes/no)."""
+
+    VERSION = datasets.Version("1.1.0")
+
+    BUILDER_CONFIGS = [
+        datasets.BuilderConfig(
+            name="posts_labeled", version=VERSION, description="This part of my dataset covers a first domain"
+        ),
+        datasets.BuilderConfig(
+            name="posts_unlabeled", version=VERSION, description="This part of my dataset covers a second domain"
+        ),
+        datasets.BuilderConfig(
+            name="articles", version=VERSION, description="This part of my dataset covers a second domain"
+        ),
+    ]
```

Comment:
can you give proper description of the different configs please ?

Reply:
oops, forgot about that! will do

Reply:
oops, fixed!